web/frontend/tsconfig.json

```diff
@@ -0,0 +1,26 @@
+{
+  "compilerOptions": {
+    "target": "es5",
+    "lib": [
+      "dom",
+      "dom.iterable",
+      "esnext"
+    ],
+    "allowJs": true,
```

Comment:
can't this be set to `false` now ?

Reply:
is this compatible with:
 
  "compilerOptions": {
    "paths": {
      "@dedis/kyber": [
        "../cothority/external/**js**/kyber/src"